core/models/tool_configuration.rb

```diff
@@ -27,6 +27,18 @@ def self.load
     return ToolConfiguration.new
   end
 
+  # Load license file from the disk
+  # @param file_name [String] name of the license file
+  # @return [String] read from the file
```

Comment:
Method will return nil in case file was not found. Please document this behavior.